web-client/views/upload.jade

```diff
@@ -245,11 +250,11 @@ html
               a(href='#' class='dropdown-toggle' data-toggle='dropdown') Help
               ul(class='dropdown-menu' role='menu')
                 li
-                  a(href='http://dondi.github.io/GRNsight/documentation.html#gettingStarted', target='_blank') Getting Started
+                  a(href='https://dondi.github.io/GRNsight/documentation.html#gettingStarted', target='_blank') Getting Started
                 li
                   a(href='https://github.com/dondi/GRNsight/wiki', target='_blank') GRNsight Wiki
                 li
-                  a(href='http://dondi.github.io/GRNsight/about.html', target='_blank') About GRNsight
+                  a(href='https://dondi.github.io/GRNsight/about.html', target='_blank') About GRNsight
```

Comment:
These ones can't be just `//`s?

Reply:
They can, I figured to maintain consistency I would change it to HTTPS

Reply:
Using `//` strikes me as being more consistent, because it will use the protocol of the current page. But not a big deal either way as these are just convenience links.